templates/scenarios/sso-tab-js/.vscode/tasks.json

```diff
@@ -31,7 +31,7 @@
             "type": "teamsfx",
             "command": "provision",
             "args": {
-                "configFile": "${workspaceFolder}/teamsfx/app.local.yml",
+                "template": "./teamsfx/app.local.yml",
```

Comment:
Why changed to this style?
`./xxx` or `../xxx`?

Reply:
`${workspaceFolder}` does not support in background task. Use relative path is more safe.

Reply:
Then we need to clearly tell what's the base path - the project folder or .vscode/ folder or anything else?
How do we treat the args path in original `teamsfx:debug-npm-install` task?

Reply:
May be `${workspaceFolder}/xxx` is better. I will change the template back.
The resolve path logic will still support relative path.